backend_new/prisma/seed-helpers/user-factory.ts

```diff
@@ -1,11 +1,15 @@
 import { Prisma } from '@prisma/client';
+import { randomAdjective, randomNoun } from './word-generator';
 
 export const userFactory = (
   roles?: Prisma.UserRolesUncheckedCreateWithoutUserAccountsInput,
+  firstName?: string,
+  lastName?: string,
+  email?: string,
```

Comment:
The format that I have been doing with these optional params in the factory has been making them an object.
```
optionalParams?: {
    roles?: Prisma.UserRolesUncheckedCreateWithoutUserAccountsInput;
    firstName?: string;
    lastName?: string;
    email?: string;
  }
```

That way when calling the factory it is easier to know what field is getting set `userFactory({firstName: "first", email: "example@example.com"})`

Reply:
updated!

Reply:
@YazeedLoonat and @ludtkemorgan, is this conversation resolved?